### DISCUSSION TITLE
Enhance item variables to take negative values.

### DISCUSSION THREAD

**Discussion by WehrJ** [AscensionGameDev/Intersect-Engine#1534] [Ideas]
The ability to put in negative values for item variables (stat bonuses, hp/mp regen, effects) would vastly open up avenues for unique itemization. I.E. lose 1% hp per tick, or lose life per hit.

![image](https://user-images.githubusercontent.com/31015361/148249827-4eca1413-6b97-447b-8b2e-df5776d4d9ba.png)

**Comment by WeylonSantana** [comment]
I've been digging through spells these days and I can say that armor effects with negative status are interesting to apply to a game

Cooldown reduction - Increases the cooldown duration of spells used
Luck - Items take much longer to drop
Exp - The exp received is cut off by the configured percentage
Tenacity - The duration of effects applied to you will last longer

A note about tenacity, if you cast a spell on yourself, if it has negative effects like stun, snare, etc, then tenacity with negative effect will make it last longer, even if the spell has the friendly checkbox. Tenacity is an effect that should only work for spells that don't have friendly checked, as they are the ones we're sure was applied by another player/npc

I have not tested the negative effects of lifesteal yet

With manasteal almost ready to enter the engine is also something to think about

Given the facts, I think it would be interesting.

About negative regeneration I don't know how it would work
